build.py

```diff
@@ -28,6 +30,22 @@
 
 
 def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
```

Comment:
feels very natural to just pass them in argv, does having them flagged help? (no real complaint, just curious)

Reply:
well, the `configs` argument is "positional", only the -j is optional (I guess that's what you mean by "flagged")